itk_pretix.info.yml

```diff
@@ -1,7 +1,7 @@
 name: 'ITK Pretix'
 type: module
 description: 'A module using pretix API to generate events from field values'
-core_version_requirement: ^8 || ^9
+core_version_requirement: ^8 || ^9 || ^10
```

Comment:
```suggestion
core_version_requirement: ^10
```

Reply:
Changed to run on d9 or d10
This way the module can be updated seperately from drupal